.env.example

```diff
@@ -155,4 +155,5 @@ LDAP_TIME_LIM=600
 IMPORT_TIME_LIMIT=600
 IMPORT_MEMORY_LIMIT=500M
 REPORT_TIME_LIMIT=12000
+REQUIRE_SAML=false
```

Comment:
Should I put any note here that says, "Don't change this unless you know what you are doing?"

Or, to reduce support requests, would it be better to leave this variable out of the example, and just put it in the documentation?

Reply:
I think it's okay to leave it as is. Nobody reads the documentation anyway lol